test/games/strategy/triplea/delegate/LHTRTest.java

```diff
@@ -227,6 +227,48 @@ public Object invoke(final Object proxy, final Method method, final Object[] arg
     // Changed to match StrategicBombingRaidBattle changes
     assertEquals(PUsBeforeRaid - 8, PUsAfterRaid);
   }
+
+  @Test
+  public void testBombingRaid2targets() {
+    final Territory germany = m_data.getMap().getTerritory("Germany");
```

Comment:
DRY violation with this method. Hard to tell what is actually new. The test is also not going to be independent from other tests, so we get into that situation where many tests fail due to the same problem.